ddtrace/contrib/influx/patch.py

```diff
@@ -0,0 +1,98 @@
+import influxdb  # version >=5.0
+import wrapt
+from influxdb.exceptions import InfluxDBClientError, InfluxDBServerError
+
+from . import metadata
+
+from ...utils.wrappers import unwrap
+from ...compat import urlencode
+from ...pin import Pin
+from ...ext import http, db, AppTypes
+from ... import config
+from ...utils.formats import get_env
+
+
+DEFAULT_SERVICE = 'influxdb'
+ROOT_SPAN = 'influx.request'
+SPAN_TYPE = 'sql'
```

Comment:
`SPAN_TYPE` should be `'influx'`.

This will be used later on when we add quantization support for influx to the trace-agent.

Reply:
See comment below. We actually want to keep it as `'sql'`, until we add proper support to the trace-agent.